### PR TITLE
- commitsWIthDefect, remove if they are included anywhere else, movin…

### PR DESCRIPTION
**Pull request by pjohnsoninfusion**
…g to the invalid section will be handed in branch 'add-commitWD-to-invalid-issues-section'

- use builders
- cleanup warnings

### CODE REVIEW

File: src/main/java/com/infusion/relnotesgen/ReleaseNotesModelFactory.java [hunk 1]
```diff
@@ -54,21 +54,39 @@ public ReleaseNotesModel get() {
...
+		        .knownIssues(knownIssuesWithFixedIssuesRemoved).releaseVersion(versionInfoProvider.getReleaseVersion())
+		        .commitTag1(gitInfo.commitTag1).commitTag2(gitInfo.commitTag2).commitsCount(gitInfo.commits.size())
+		        .gitBranch(gitInfo.gitBranch).configuration(configuration).errors(errors)
+                .build();
 
         return model;
     }
 
-	private ImmutableSet<ReportJiraIssueModel> removeFixedIssuesFromKnownIssues(final ImmutableSet<ReportJiraIssueModel> knownIssues, 
+	private ImmutableSet<ReportCommitModel> filterOutJiraIssues(ImmutableSet<ReportCommitModel> commitsWithDefectIds,
```
Comment (karpikpl) [inline src/main/java/com/infusion/relnotesgen/ReleaseNotesModelFactory.java]: Method not used?

Comment (pjohnsoninfusion) [inline src/main/java/com/infusion/relnotesgen/ReleaseNotesModelFactory.java]: It's just a weird diff result.  Both methods are there.
image

**Comment by karpikpl** [comment]
so I was talking about **filterOutJiraIssues**
is this used anywhere?

**Comment by pjohnsoninfusion** [comment]
Good eye, I forgot to uncomment out the usage of it.

**Pull request merged**